### PR TITLE
Iced 0.13

### PR DESCRIPTION
**Pull request by wash2**

### CODE REVIEW

**Comment by wiiznokes** [comment]
Looks like `Command` is not available anymore ?
screenshot-2024-10-18-15-47-07

**Comment by mmstick** [comment]
@wiiznokes They're called Tasks now

**Comment by wiiznokes** [comment]
Ah yeah, i forgot thanks

**Review by mmstick — DISMISSED** [review]

**Comment by mmstick** [comment]
Noticing a lot of these logs whenever any redraw happens.

```
2024-10-18T15:57:50.175716Z  INFO wgpu_core::device::resource: Device::maintain: waiting for submission index 133
```

File: src/widget/toggler.rs
Comment (wiiznokes) [inline src/widget/toggler.rs:1]: Could this file be removed, and the helper from Iced exported instead ? (to allow a disabled state)

Comment (mmstick) [reply src/widget/toggler.rs:1]: Just needs to remove the requirement on supplying an on_toggle function.

**Comment by wash2** [comment]
> wgpu_core::device::resource: Device::maintain: waiting for submission index

This seems to be an issue with the 22.0 release of wgpu, that has since been reduced to a trace, and moved into the if statement. https://github.com/gfx-rs/wgpu/blob/278d278b28ff7b424dc7dde0c0d7519e8d839ec1/wgpu-core/src/device/resource.rs#L446
Fixed by: https://github.com/gfx-rs/wgpu/commit/841e7c885713983a354a814e5635bf43c1e37f56

**Review by mmstick — APPROVED** [review]

**Comment by Adam-Cosner** [comment]
cosmic::theme and cosmic::style need to be updated to reflect the new 0.13 style system

**Pull request merged**